specification/validated-relationships-service-api.yaml

```diff
@@ -1,5 +1,5 @@
 # This is an OpenAPI Specification (https://swagger.io/specification/)
-# for validated-relationships-service-api owned by NHS Digital (https://digital.nhs.uk/)
+# for validated-relationships owned by NHS Digital (https://digital.nhs.uk/)
```

Comment:
Not sure about this change as I don't know if there are a knock on effects to the documentation of our API. Not sure if in the developer portal it's `validated-relationships` or `validated-relationships-service-api`

Reply:
@JackPlowman this is fine. Good spot.